ts/IdentificationModal.tsx

```diff
@@ -128,18 +137,140 @@ class IdentificationModal extends React.PureComponent<Props, State> {
     };
   }
 
+  public componentDidMount() {
+    AppState.addEventListener("change", this.appStateChangeListener.bind(this));
```

Comment:
this event is already subscribed by the `RootComponent` and it's used to update the [redux state](https://github.com/teamdigitale/io-app/blob/eea5b9399746a108b198e010e9194ef4a001a7af/ts/store/reducers/appState.ts#L27) - I think it would be better to just connect this component to the redux state. - @fpersico what do you think?

Reply:
changed.